github-dark.css

```diff
@@ -1529,8 +1529,9 @@
   a.filter-item.selected, .date-selector td.selected,
   .date-selector td.selectable-day:hover, .subnav-item:hover, .subnav-item:focus,
   .owner-select-target:hover, .owner-select-target:focus,
-  .suggester li.navigation-focus, .suggester li.navigation-focus small {
-    color: #efefef !important;
+  .suggester li.navigation-focus, .suggester li.navigation-focus small,
+  .label-select-menu .select-menu-item.selected, .octicon .octicon-x {
```

Comment:
All octicons (which are SVG) on the site have `fill: currentColor`, so you just need to set `color` on any parent element to color them too. If there's no wrongly colored parent in the way, that icon should get it's color inherited from `.select-menu-item.selected` ideally.